tests/functional/olp-cpp-sdk-dataservice-read/VersionedLayerClientTest.cpp

```diff
@@ -118,4 +195,182 @@ TEST_F(VersionedLayerClientTest, GetPartitions) {
   EXPECT_TRUE(mock_server_client_->Verify());
 }
 
+TEST_F(VersionedLayerClientTest, GetAggregatedData) {
+  olp::client::HRN hrn(kTestHrn);
+
+  constexpr auto kTileId = "23606938";
+  constexpr auto kLayer = "testlayer";
+  constexpr auto kQuadTreeDepth = 4;
+  const auto root_tile = olp::geo::TileKey::FromHereTile(kTileId);
+  const auto tile = root_tile.ChangedLevelTo(16);
```

Comment:
Platform does not support level 16, max level is 14 or 15. Why do you test with 16?

Reply:
I didn't know that :-)
Any level below 32 is valid for TileKey, so I was thinking level 16 is fine.
Fixed test to use level 15 instead.